lib/Target/Lattigo/LattigoEmitter.cpp

```diff
@@ -303,7 +361,9 @@ LogicalResult LattigoEmitter::printNewMethod(::mlir::Value result,
 LogicalResult LattigoEmitter::printEvalInplaceMethod(
     ::mlir::Value result, ::mlir::Value evaluator, ::mlir::Value operand,
     ::mlir::Value operandInplace, std::string_view op, bool err) {
-  std::string errResult = err ? ", err := " : "";
+  // TODO: err := or err = depends on the context...
```

Comment:
you could avoid this by generating new variable names for each err. Just have an incrementing counter class global.